tests/PR_testing/install.sh

```diff
@@ -72,6 +71,9 @@ if [ ! -d "${_ROOT_INSTALLATION_DIR}" ]; then
     ansible-galaxy collection install ansible.netcommon
     ansible-galaxy collection install arista.eos
 
+    echo "install molecule package"
+    pip install molecule
```

Comment:
I would suggest installing all dev requirements -> https://github.com/aristanetworks/ansible-cvp/blob/devel/ansible_collections/arista/cvp/requirements-dev.txt - since we may also control the version. And no harm in having some of the other tools for testing.
```suggestion
    pip install -r ${_LOCAL_CVP}/ansible_collections/arista/cvp/requirements-dev.txt
```